lld/COFF/SymbolTable.cpp

```diff
@@ -245,14 +245,23 @@ void SymbolTable::reportUndefinedSymbol(const UndefinedDiag &undefDiag) {
 }
 
 void SymbolTable::loadMinGWSymbols() {
+  std::vector<Symbol *> undefs;
   for (auto &i : symMap) {
```

Comment:
So the problem we're fixing here is that we avoid mutating `symMap` while iterating over it?